css/style.css

```diff
@@ -1,52 +1,330 @@
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	vertical-align: baseline;
-}
-
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    vertical-align: baseline;
+}
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+    display: block;
 }
 
 body {
-	line-height: 1;
+    line-height: 1;
 }
 
-ol, ul {
-	list-style: none;
+ol,
+ul {
+    list-style: none;
 }
 
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+    quotes: none;
 }
 
-blockquote::before, blockquote::after,
-q::before, q::after {
-	content: '';
-	content: none;
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+    content: "";
+    content: none;
 }
 
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+    border-collapse: collapse;
+    border-spacing: 0;
 }
 
 /* Do not write a css rule above here! Use the space below  */
 
+/* Write your css below here */
+
+.container {
+    min-height: 100vh;
+    max-width: 100vw;
+    box-sizing: border-box;
+    border: white solid 2px;
+}
+
+header {
+    background-color: rgb(102 189 102);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 30px;
+}
+
+.header-img-container img {
+  width: 200px;
+  height: auto;
+  transition: all 0.5s ease-in;
+}
+
+.content-image img {
+    width: 100%;
+    height: 100%;
+}
+
+.header-img-container img:hover {
+    transform: scale(1.2);
+}
+
+header nav {
+    padding-top: 30px;
+}
+
+header nav a {
+    margin-right: 20px;
+    font-size: 15px;
+    text-decoration: none;
+    color: wheat;
+  }
+  
+  header nav a:hover {
+    color: rgb(230 168 53);
+  }
+  
+  .heading {
+    background-color: #fff;
+    color: rgb(146 96 2);
+    padding: 20px 15px;
+    font-size: 40px;
+    cursor: pointer;
+}
+
+main {
+    background-color: green;
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 20px;
+    gap: 10px;
+}
+
+.content {
+  max-width: 410px;
+  height: 300px;
+  background-color: wheat;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  transition: all 1s;
+}
+
+.content:hover {
+  transform: rotate(360deg);
```

Comment:
Something seems to be faulty here. The animation is not seamless and affects multiple items at the same time. Kindly fix this